### PR TITLE
Plugin proposal and design sketch

### PR DESCRIPTION
**Pull request by mihaibirsan**
Please have a read and give me feedback before I set this loose. :-)

Quick previews:
- https://github.com/NimbleNexus/obsidian-github-comments/blob/9d42f95bd0e9c7f9c743f56fb260a4f3040eb962/README.md
- https://github.com/NimbleNexus/obsidian-github-comments/blob/9d42f95bd0e9c7f9c743f56fb260a4f3040eb962/DEVELOPMENT.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,96 +1,51 @@
...
-```
-
-## API Documentation
-
-See https://github.com/obsidianmd/obsidian-api
+# Obsidian GitHub Comments
+
+Context: Obsidian users have limited options to collaborate and get early feedback on their notes. We propose, seek feedback and funding for a plugin that leverages GitHub's comment system.
+
+Examples:
```
Comment (vlad-diac) [inline README.md:5]: You know the obsidian user base the best, but I feel like getting some estimated user numbers would make the case here.

File: README.md [hunk 2]
```diff
@@ -1,96 +1,51 @@
...
+- Reply to GitHub comments
+- "Add GitHub comment" command
+- Features to broaden the niche
+  + Make it a single step to backup notes to GitHub _(given a GitHub account, created and signed into separately, in a browser)_
+  + Make it a single step to open a GitHub vault
+- Extended features
+  + View most recent comments and replies across the entire vault
+  + Keep track of comments across branches _(power user feature)_
+  + Comment on notes published as HTML
+    - Note: There's an opportunity here for Obsidian Publish to support GitHub comments for certain vaults.
```
Comment (vlad-diac) [inline README.md:28]: This is vague for someone who doesn't know about Obsidian Publish.

File: README.md [hunk 3]
```diff
@@ -1,96 +1,51 @@
...
+
+Alternatives to consider:
+- Make comments collaboration work with alternative services <a name="^202310201601"></a>
```
Comment (vlad-diac) [inline README.md:31]: Can you give some examples?

File: README.md (hunk 2 above)
Comment (mihaibirsan) [reply README.md:28]: That line is written specifically for the people who do know about Obsidian Publish.

I will take this as an opportunity to expand later.

File: README.md (hunk 1 above)
Comment (mihaibirsan) [reply README.md:5]: There are a few estimates that I can pull together. However, for this announcement we're targeting Obsidian users specifically... All they need to be convinced is whether or not we're solving a problem they have.

**Pull request merged**